lib/src/plaform/open_file.dart

```diff
@@ -26,10 +26,13 @@ class OpenFile {
       if (Platform.isMacOS) {
         _result = mac.system(['open', '$filePath']);
       } else if (Platform.isLinux) {
+        var filePathLinux = Uri.file(filePath!);
         if (linuxByProcess) {
-          _result = Process.runSync('xdg-open', [filePath!]).exitCode;
+          _result = Process.runSync('xdg-open', ['${filePathLinux.toString()}'])
```

Comment:
no string interpolation needed

Reply:
@mx1up it's not really necessary, I just used it to make it easier to understand.